Makefile

```diff
@@ -286,6 +286,12 @@ start-e2e-import-mainnet-test: e2e-images
 	export ZETACORED_START_PERIOD=15m && \
 	cd contrib/localnet/ && ./scripts/import-data.sh mainnet && $(DOCKER_COMPOSE) up -d
 
+start-e2e-consensus-test: e2e-images
```

Comment:
How does just this change allow to test for consensus break?

Reply:
Because you're running the `develop` version with your locally built/PR version. If a block is computed differently, the network will halt.

Right?